packages/core-mobile/app/services/nft/types.ts

```diff
@@ -11,13 +11,21 @@ export type NftImageData = {
   type: NftContentType
 }
 
+export enum NftLocalStatus {
+  Unprocessed = 'Unprocessed',
+  Processed = 'Processed',
+  Unprocessable = 'Unprocessable'
+}
```

Comment:
nice add! in the future, we can use this to further improve our processing logic. we can skip reprocessing already processed nfts for example.